src/Twig/NetworkLogoExtension.php

```diff
@@ -69,10 +44,33 @@ public function getNetworkLogo(string $nid, string $size): string
             $size = '112x63';
         }
 
-        $logo = 'bbc';
-        if (in_array($nid, self::ALLOWED_LOGOS)) {
-            $logo = $nid;
+        $path = ['images/', 'logos/'];
+        $logo = ($this->isAllowedLogo($nid)) ? $nid : 'bbc';
+
+        if ($this->isSvgLogo($nid)) {
+            array_push($path, 'svg/', $logo, '.svg');
+        } else {
+            array_push($path, $size, '/', $logo, '.png');
         }
-        return $this->packages->getUrl('images/logos/' . $size . '/' . $logo . '.png');
+
+        return $this->packages->getUrl(implode('', $path));
+    }
+
+    private function getParameter($param)
+    {
+        return $this->params->get($param);
+    }
+
+    private function isSvgLogo($nid): bool
+    {
+        $branding = $this->getParameter('branding');
+        return in_array($nid, $branding['logos']['filetypes']['svg']);
```

Comment:
```suggestion
        return \in_array($nid, $branding['logos']['filetypes']['svg']);
```
enable some opcode optimization, details and bechmark if you feel curiosity https://github.com/Roave/FunctionFQNReplacer